### PR TITLE
Fix create docker app

### PR DESCRIPTION
**Pull request by carabasdaniel**

### CODE REVIEW

**Comment by florindragos** [comment]
👍 Tested this fix and it works.
We should have a separate test for docker apps, maybe even one that actually starts an app from a docker image.

File: src/CloudFoundry.CloudController.Test.Integration/DockerTest.cs [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+            }
+            Assert.IsNotNull(routeResponse);
+
+            try
+            {
+                assocResponse = client.Routes.AssociateAppWithRoute(routeResponse.EntityMetadata.Guid, dockerApp.EntityMetadata.Guid).Result;
+            }
+            catch (Exception ex)
+            {
+                Assert.Fail("Errot binding route to app: {0}", ex.ToString());
```
Comment (florindragos) [inline src/CloudFoundry.CloudController.Test.Integration/DockerTest.cs]: typo

**Comment by florindragos** [comment]
verify this please

**Comment by carabasdaniel** [comment]
Fixed typo

**Comment by mihaibuzgau** [comment]
verify this please

**Comment by carabasdaniel** [comment]
verify this please

**Comment by carabasdaniel** [comment]
verify this please

**Pull request merged**